### PR TITLE
Add documentation.

### PR DESCRIPTION
**Pull request by vcarpani**
This PR adds an asciidoc document that can be compiled as html

Below you can find an example of the rendered html:

### CODE REVIEW

**Comment by vcarpani** [comment]
image
image
image

**Comment by de-vri-es** [comment]
Maybe add a script to generate the HTML with the same flags we use in the RPC doc generator?

**Comment by vcarpani** [comment]
> Maybe add a script to generate the HTML with the same flags we use in the RPC doc generator?

I cannot find any flags for the current rpc doc generator, shall we have a short discussion on this?

File: documentation/fizyr-rpc.adoc [hunk 1]
```diff
@@ -0,0 +1,276 @@
+= Fizyr RPC
```
Comment (de-vri-es) [inline documentation/fizyr-rpc.adoc:1]:
```suggestion
:sectanchors:
:sectlinks:
:toc:
= Fizyr RPC
```

**Comment by vcarpani** [comment]
@de-vri-es shall we merge this?

**Pull request merged**